modules/timedrift/timedrift.go

```diff
@@ -25,14 +27,58 @@ import (
 )
 
 type module struct {
+	SandboxProfile sandbox.SandboxProfile
 }
 
 func (m *module) NewRun() modules.Runner {
 	return new(run)
 }
 
+func (m *module) GetSandboxProfile() sandbox.SandboxProfile {
+	return m.SandboxProfile
+}
+
 func init() {
-	modules.Register("timedrift", new(module))
+	m := new(module)
+	sandbox := sandbox.SandboxProfile{
+		DefaultPolicy: seccomp.ActTrap,
+		Filters: []sandbox.FilterOperation{
+			sandbox.FilterOperation{
+				FilterOn: []string{
+					"select",
+					"futex",
+					"write",
```

Comment:
I think you need write in order to write the results back on the pipe and also for our sandboxing code to print the error message back. We'll try to see if we can filter it to be only to fd 1 and 2.